client/src/Context.tsx

```diff
@@ -3,14 +3,16 @@ import { createContext } from 'react';
 type State = {
   currentProjectId: number;
   userId: string;
-  isModalOpen: boolean;
+  isBugOpen: boolean;
```

Comment:
personally I'd leave this as isBugModalOpen, as isBugOpen could have a wider range of meanings